website/static/files/build_orca_reconcile.sql

```diff
@@ -6,150 +6,141 @@ SET search_path TO orca, public;
-    , last_update     timestamp with time zone NOT NULL
-    , end_time        timestamp with time zone NULL
-    , error_message   text NULL
-    , CONSTRAINT PK_reconcile_jobs_1 PRIMARY KEY(id)
-    , CONSTRAINT CHECK_reconcile_jobs_status CHECK status in ('pending', 'getting S3 list', 'finding orphans', 'finding mismatches', 'error', 'complete')
-    );
+  CREATE TABLE reconcile_status  ( 
+    id   	int2 NOT NULL,
+    value	text NOT NULL,
+    CONSTRAINT PK_orca_status PRIMARY KEY(id),
+    CONSTRAINT UNIQUE_reconcile_status_value UNIQUE (value),
+    CONSTRAINT CHECK_reconcile_jobs_status CHECK status in ('pending', 'getting S3 list', 'finding orphans', 'finding mismatches', 'error', 'complete')
+  );
+  COMMENT ON TABLE reconcile_status IS 'Refrence table for valid status values and status order.';
+  COMMENT ON COLUMN reconcile_status.id IS 'Status ID';
+  COMMENT ON COLUMN reconcile_status.value IS 'Human readable status value';
 
-    COMMENT ON TABLE reconcile_jobs
-      IS 'Manages internal reconciliation job information.';
-    COMMENT ON COLUMN reconcile_jobs.id
-      IS 'Job ID unique to each internal reconciliation job.';
-    COMMENT ON COLUMN reconcile_jobs.status
-      IS 'Current status of the job.';
-    COMMENT ON COLUMN reconcile_jobs.start_time
-      IS 'Date and time the internal reconcile job started.';
-    COMMENT ON COLUMN reconcile_jobs.last_update
-      IS 'Date and time the job status was last updated.';
-    COMMENT ON COLUMN reconcile_jobs.end_time
-      IS 'Time the job completed and wrote the report information.';
-    COMMENT ON COLUMN reconcile_jobs.error_message
-      IS 'Critical error the job ran into that prevented it from finishing.';
 
+  ALTER TABLE reconcile_status
+    ADD CONSTRAINT UNIQUE_reconcile_status
+    UNIQUE (value);
 
-    -- Create reconcile_s3_objects table
-    CREATE TABLE IF NOT EXISTS reconcile_s3_objects
-    (
-      job_id                  int8 NOT NULL
-    , orca_archive_location   text NOT NULL
-    , key_path                text NOT NULL
-    , etag                    text NOT NULL
-    , last_update             timestamp with time zone NOT NULL
-    , size_in_bytes           int8 NOT NULL
-    , storage_class           text NOT NULL
-    , CONSTRAINT PK_reconcile_s3_objects_1 PRIMARY KEY(job_id,orca_archive_location,key_path)
-    , CONSTRAINT reconcile_jobs_s3_objects_fk FOREIGN KEY(job_id) REFERENCES reconcile_jobs(id)
-    );
+  -- Create reconcile_jobs table
+  CREATE TABLE reconcile_jobs  ( 
+    id                     	int8 NOT NULL,
+    orca_archive_location  	text NOT NULL,
+    status_id              	int2 NOT NULL,
+    inventory_creation_time	timestamp with time zone NOT NULL,
+    start_time             	timestamp with time zone NOT NULL,
+    last_update            	timestamp with time zone NOT NULL,
+    end_time               	timestamp with time zone NULL,
+    error_message          	text NULL,
```

Comment:
error_message is not available in reconcile_jobs table in the architecture drawing.

Reply:
Resized.